Sources/Signal+SwiftConcurrency.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Signal+SwiftConcurrency.swift
+//  ReactiveSwift
+//
+//  Created by Marco Cancellieri on 2021-11-11.
+//  Copyright (c) 2021 GitHub. All rights reserved.
+//
+#if compiler(>=5.5) && canImport(_Concurrency)
+import Foundation
+
+@available(macOS 12, iOS 15, watchOS 8, tvOS 15, *)
```

Comment:
With Xcode 13.2 (currently in beta), Swift Concurrency has been back ported to iOS 13 and macOS 10.15 so if we wait for that we might be able to bring this availability check down:

```
#if compiler(>=5.5.2) && canImport(_Concurrency)
@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
```

Reply:
Good point, might be worth the wait!